test/views/declaration/spec/ViewMatchers.scala

```diff
@@ -248,10 +248,16 @@ trait ViewMatchers {
     )
 
   def haveFieldErrorLink(fieldName: String, link: String): Matcher[Element] =
-    new ContainElementWithIDMatcher(s"error-message-$fieldName-input") and new ElementHasAttributeValueMatcher(
-      "href",
-      link
-    )
+    new Matcher[Element] {
+      override def apply(left: Element): MatchResult = {
+        val element = left.getElementById(s"$fieldName-error")
+        MatchResult(
+          element != null && element.attr("href") == link,
+          s"View not contains $fieldName with $link",
```

Comment:
Maybe "Information about error in field with link"

Reply:
Not sure about it, "Information about error in field with link" doesn't tell what happened